src/elements/forms/styles.css

```diff
@@ -108,10 +110,63 @@
   transform: translateY(0);
 }
 
+/**
+ * FormLabel
+ * =============================================================================
+ */
+
+.label {
+  @extend %label;
+}
+
+/**
+ * FormControl
```

Comment:
Maybe you should use regions comments. It allows the collapsing of the whole region at once. Not sure if it is supported in your editor I hope so. :) 

/* region: FormControl */
code here
/* endregion */

Reply:
I am not a fan of such comments. If you do not mind, I will not add it.

Personally, I prefer having it in the separate files but we need to explore CSS Modules config more deeply to make hashes be the same for all files within the directory. I know the way to do this but I do not think it's a need at the moment.